miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/ManifestApi.kt

```diff
@@ -3,12 +3,14 @@ package com.rakuten.tech.mobile.miniapp.api
 import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Path
+import retrofit2.http.Query
 
 internal interface ManifestApi {
-    @GET("miniapp/{miniappId}/version/{versionId}/manifest")
+    @GET("host/{hostappId}/miniapp/{miniappId}/version/{versionId}/manifest")
```

Comment:
This one is still missing the `@Path` parameter for `hostappId`.

Reply:
My bad..! Thanks..!